### DISCUSSION TITLE
Is the "beignet-opencl-icd" package no longer available in 22.04?

### DISCUSSION THREAD

**Discussion by rcv11x** [tteck/Proxmox#1119] [Q&A]
hi all, i installed jellyfin through tteck and everything went very well, i have had it running for a while and installed the ubuntu 20.04 version but now i have switched from ubuntu 20.04 to 22.04 and before i had the packete `beignet-opencl-icd` and now it is not available, it is supposed to be for tone mapping right? is there an alternative? i was told to install the `intel-opencl-icd` package.

**Comment by tteck** [comment]
Correct,  Beignet is not available in Ubuntu 22.04 and newer.

While both Beignet and Intel OpenCL ICD provide OpenCL implementations for Intel processors, Beignet is specifically focused on providing a high-performance OpenCL implementation for Intel graphics devices, while Intel OpenCL ICD is designed to support Intel processors across a range of platforms.

The `intel-opencl-icd` package has not been personally tested by me, but I can include it in the script for Ubuntu 22.04 and newer versions.

**Comment by tteck** [comment]
Please provide feedback on the functionality of the Intel OpenCL ICD implementation in the script, which has been added for Ubuntu 22.04 and newer versions.

**Reply by rcv11x** [reply]
But what exactly is this package for? Because I can still transcode without problem with intel-opencl-icd, is it for tone mapping or something like that?

**Reply by tteck** [reply]
Beignet and Intel OpenCL ICD both provide OpenCL implementations for Intel processors. Use one or the other.
